core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackPosition.java

```diff
@@ -0,0 +1,30 @@
+package de.danoeh.antennapod.core.util.playback;
+
+/**
+ * Data object for playback possition
+ */
+public class PlaybackPosition {
```

Comment:
Why wrap this in a class? Combining position and last-played time stamp is not really intuitive to me. I think they are pretty much independent things.

Reply:
Reason for this is that some methods take now 3-4 arguments, which goes against Clean Code. I'm only getting to know code base while doing this, i tried to group some arguments together that suits specific purpose. If you think that this class doesn't make sens, or could be named more accurate - let me know. I prefer wrapping arguments together when they create common entity.

Reply:
Generally, I agree that wrapping things increases the readability. In this case, though, I do not think it is logical to wrap them. A position is something completely different to a real-world timestamp. The two attributes are also used for two different things in the UI. I would prefer the non-wrapped version here :)